src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpUseImplicitTypeDiagnosticAnalyzer.cs

```diff
@@ -274,4 +251,24 @@ protected override bool ShouldAnalyzeDeclarationExpression(DeclarationExpression
             return base.ShouldAnalyzeDeclarationExpression(declaration, semanticModel, cancellationToken);
         }
     }
+
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    internal sealed class CSharpUseImplicitTypeDiagnosticAnalyzer : CSharpTypeStyleDiagnosticAnalyzerBase
+    {
+        private static readonly LocalizableString s_Title =
```

Comment:
is s_CapLetter the naming style for statics?

Reply:
No idea. This is existing code.

---
In reply to: [172284826](https://github.com/dotnet/roslyn/pull/25208#discussion_r172284826) [](ancestors = 172284826)